.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
This file looks like a very specific file for your particular editor (which one ?)
We should probably not include this here, otherwise we end up with a bunch of files for everyone's favorite editor. Probably good to keep it in your local check-out for your edits.